app/fastTrack/src/appRegistry/FastTrackBanner.js

```diff
@@ -1,20 +1,98 @@
-// @flow strict
+// @noflow
```

Comment:
Could you explain the reason behind this?

Reply:
I will fix everything. I didn't want to spend a lot of time on this component since I will delete everything anyways.